packages/components/src/templates/next/components/complex/DynamicDataBanner/DynamicDataBannerClient.tsx

```diff
@@ -59,15 +64,19 @@ const DynamicDataBannerUI = ({
   DynamicDataBannerClientProps,
   "title" | "label" | "url" | "LinkComponent" | "errorMessageBaseParagraph"
 > & {
-  data: { label: string; value: string }[]
+  data: { label: string; value: string | undefined }[]
```

Comment:
what's the `label` when `value === undefined`?

Reply:
`label` is defined in the component

Reply:
slight refactor https://github.com/opengovsg/isomer/pull/980/commits/fe5ba6429bbcc3bdc5c4c39b5263a72e3d69579c to replace `value: string | undefined` with `value?: string`

Reply:
yup! label is what the user want to display, and the value will only be known after the data is fetched